agendas/2025/04-Apr/17-community-working-group-april2-2025

```diff
@@ -0,0 +1,41 @@
+| This is an open meeting: To attend, edit and PR this file. (Edit: ✎ above, or press "e") |
+| ---------------------------------------------------------------------------------------- |
+
+# Community Working Group — April 2025 - Website
+
+- **Date & Time**: [April 17, 2025, 9:00 – 10:00 AM PDT](https://www.timeanddate.com/worldclock/converter.html?iso=20250417T160000&p1=224&p2=179&p3=136&p4=268&p5=367&p6=438&p7=248&p8=240)
```

Comment:
Note that this clashes with the composite schemas WG; I'd recommend hosting it either an hour later or 2 hours earlier - see the calendar linked on line 15 for scheduled events on this day.

Reply:
https://calendar.google.com/calendar/embed?src=linuxfoundation.org_ik79t9uuj2p32i3r203dgv5mo8%40group.calendar.google.com